src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
```

Comment:
Lots of unrelated whitespace changes up here?

Reply:
@333fred That's a bug in whatever tool generates this file when editing in Visual Studio. It *sometimes* for some unknown reason adds the exact same whitespaces. 😕 

Will clean it up.